samples/python/image_generation/README.md

```diff
@@ -20,6 +20,10 @@ Users can change the sample code and play with the following generation paramete
 - Apply multiple different LoRA adapters and mix them with different blending coefficients
 - (Image to image and inpainting) Play with `strength` parameter to control how initial image is noised and reduce number of inference steps
 
+> [!NOTE]  
+> All samples use `openvino_genai.TorchGenerator` as an argument to `generate` call to align random numbers generation with Diffusers library where `torch.Generator` is used underhood.
```

Comment:
```suggestion
> All Python samples use `openvino_genai.TorchGenerator` as an argument to `generate` call to align random numbers generation with Diffusers library where `torch.Generator` is used underhood.
```
It reads "every existing genai sample" otherwise, including C++.

Reply:
this README.md is in [samples/python/image_generation](https://github.com/openvinotoolkit/openvino.genai/pull/1379/files/66a506ce8267298a1ec7d03f777026ced7ead8cc#diff-b0ab5833a18c3e111c44b63b56dfe361d0a894180b6a522564b7da27d830d4a4)

Reply:
how C++ samples can use torch.Generator ?

Reply:
Before I red the changes my assumption was that you had copied full torch generator to genai. That would enable it for C++

Reply:
Updated, please check